data/data-pipeline/data_pipeline/etl/sources/census_acs/etl.py

```diff
@@ -41,10 +41,10 @@ def __init__(self):
 
         self.STATE_MEDIAN_INCOME_FTP_URL = (
             settings.AWS_JUSTICE40_DATASOURCES_URL
-            + "/2014_to_2019_state_median_income.zip"
```

Comment:
wondering if this should be extracted to a constant somewhere?

Reply:
For ETL purposes, those are the constants (that's why they're all CAPS) and they're set at the ETL file level because they're not used anywhere else.